src/runtime/GlobalObjectBuiltinArray.cpp

```diff
@@ -1489,6 +1489,20 @@ static Value builtinArrayShift(ExecutionState& state, Value thisValue, size_t ar
     return first;
 }
 
+static Value builtinArrayUnscopables(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+{
+    Object* blackList = new Object(state);
```

Comment:
The `blackList` object prototype should be `null`.

Reply:
@rerobika  Thank you for the reviews and for the suggestions!